ignite/cmd/ignite/main.go

```diff
@@ -12,22 +12,26 @@ import (
 )
 
 func main() {
+	os.Exit(run())
+}
+
+func run() int {
 	ctx := clictx.From(context.Background())
 
 	cmd := ignitecmd.New()
 
 	// Load plugins if any
 	if err := ignitecmd.LoadPlugins(ctx, cmd); err != nil {
 		fmt.Printf("Error while loading chain's plugins: %v\n", err)
-		os.Exit(1)
+		return 1
```

Comment:
Maybe constants could be defined. Something in the lines of:

```go
const (
    exitCodeOK = iota
    exitCodeError 1
)

func run() int {
    // ...

    return exitCodeOK
}
```

Reply:
better yes cdcff2319dddaca9d7698358ab30858bbddef3d7 !